blockchain/blockchain-wallet-v4-frontend
# SFOX homepage signup banner

## Description
Adds a reminder banner to homepage for possible SFOX users to signup and start buying BTC.  Their current setup progress should be reflected in the banner. 

## Change Type
Feature

## Testing Steps
Sign into app with a SFOX whitelisted country and state and ensure signup banner displays. Go through SFOX signup steps and ensure current steps are correctly shown on the banner.

## PR Creator Checklist

## PR Reviewer Checklist
